pkg/daemon/ceph/osd/volume.go

```diff
@@ -121,6 +121,15 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 				batchArgs = append(batchArgs, deviceArg)
 				configured++
 			} else {
+				// Reporting
+				immediateReportArgs := append(batchArgs, []string{
+					"--report",
```

Comment:
immediateReportArgs and err are redeclared below. also the reporting should include deviceArg and osdPerDeviceFlag